CHANGES.md

```diff
@@ -39,6 +39,14 @@ To be released.
  -  Added `ICryptoBackend<T>.Sign(HashDigest<T>, PrivateKey)` method.
     [[#932]]
  -  `DefaultCryptoBackend` became to `DefaultCryptoBackend<T>`.  [[#932]]
+ -  Added `ImmutableArray<byte>`-typed `actionsHash` parameter to
+    `BlockHeader` constructor. [[#931], [#935]]
+ -  Added `HashDigest<SHA256>`-typed `actionsHash` parameter to
+    `Block<T>()` constructor. [[#931], [#935]]
+ -  Added `ImmutableArray<byte>`-typed `preCommitHash` parameter to
+    `BlockHeader` constructor. [[#931], [#935]]
+ -  Added `HashDigest<SHA256>`-typed `preCommitHash` parameter to
+    `Block<T>()` constructor. [[#931], [#935]]
```

Comment:
Maybe a nullable `HashDigest<SHA256>`-typed `evaluationDigest` is also added to `Block<T>()` constructor?

Reply:
solved at https://github.com/planetarium/libplanet/pull/935/commits/2fb44dda1dff79829d89cfc497109143d4c6d462